### ISSUE TITLE
Not loading all pokemon 

### ISSUE THREAD

**Issue by Nickg1995** [AHAAAAAAA/PokemonGo-Map#2949]
I am running the latest branch against one of the versions I downloaded about a week ago and I am noticing pokemon on the old version that do not appear on the latest version. Alongside that, its very slow to scan. I have read through other issues people are encountering and I have currently have this  python runserver.py -a ptc -u ***** -p ***** -l “Location” -st 30 -L en -k A**** -P 3000 -sd 5 -t 1 -ng -nk

Despite all that its now showing as much pokemon as it did a few days ago and its much slower. Does anyone actually have a solid answer and know how to fix this?

Kind regards :)

**Comment by JackDoan** [comment]
If you're not running the develop branch, some pokemon are going to get skipped. With the 0.31 update, Niantic dropped the 'see pokemon near you' radius from 100m to 70m. Since the master branch code still uses 100m circles, some pokemon slip through the cracks, so to speak.

develop:a83ce5c0e6bfa4a97df485498e27fe3a0181ef7a, the latest commit right now, works perfectly fine, if you want to try it.

As for the slowness you're experiencing, that's because API calls have been ratelimited. Lowering `-sd` to something like 2 will probably help, but I'm not exactly sure what you'd be risking by doing that.

**Comment by ku6a** [comment]
I'm actually having a similar problem. In a way I am glad that it happened because it prompted me to do some additional research on how to circumvent the problem with the speed being slower. From what I have read I will need multiple accounts. This might be a blessing in disguise.

Cheers

**Comment by Nickg1995** [comment]
@JackDoan Thanks for the response. Yea I downloaded the developer one before but it said to run the grunt build in terminal and I am having trouble to locate where that is. Any chance you could help direct me to this :)

**Comment by JackDoan** [comment]
@Nickg1995 You're welcome!
If possible, I'd really recommend running this in Docker. It totally automates the build process and isolates it from your machine.

What operating system are you using?

**Comment by Nickg1995** [comment]
I am currently using OS X (mac). From what I am looking at, Docker is not compatible with mac :/

**Comment by prinzpi** [comment]
Is it also possible to use screen instead of docker?
Thanks.

**Comment by JackDoan** [comment]
@Nickg1995 [Try this?](https://docs.docker.com/engine/installation/mac/)

@prinzpi yes, but no. Docker isolates the server & the build process from your machine. Screen just prevents the server from exiting when you close the terminal.

**Comment by Bizadon** [comment]
@JackDoan Hi. Have the same problem as a topic starter. You say to use this: https://github.com/AHAAAAAAA/PokemonGo-Map/commit/a83ce5c0e6bfa4a97df485498e27fe3a0181ef7a

If I understand you correctly, so this 2 files in the link I need to add in the main container to the folders static/js/ and  templates/, right? And everything with the search will be fine?

**Comment by JackDoan** [comment]
@Bizadon No, I'm afraid you may be misinterpreting me. All you need to do is clone the develop branch at that point, and build the container:
`docker build -t pogo-map .`
and then run it:
`docker run --name pogo -d pogo-map \
--auth-service ptc \
--username USERNAME \
--password PASS \
--location "Dallas, TX" \
--gmaps-key randomlookingcharachtershere-use-your-key \
--display-in-console \'

**Comment by Nickg1995** [comment]
@JackDoan what you posted there, will that complete the grunt build?

**Comment by lepschay** [comment]
Tried with a fresh PTC acc from an unbanned IP with -st 5 -sd 30 -T 1 
-> No rare Pokémon... I´ll guess Niantic pulls something like for example the original gps data string from the smatphone you are supposed to be logged in from :(

**Comment by excitedbox** [comment]
@lepschay your scan delay is way too high. You will miss a lot of pokemon like that. Set it to 5 for better results.

**Comment by lepschay** [comment]
@excitedbox was just for testing if a higher scan delay helps to show rare pokemon again, but no change :(
